graphql-dgs-extended-validation/src/main/kotlin/com/netflix/graphql/dgs/autoconfig/DgsExtendedValidationAutoConfiguration.kt

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2021 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.netflix.graphql.dgs.autoconfig
+
+import com.netflix.graphql.dgs.DgsComponent
+import com.netflix.graphql.dgs.DgsRuntimeWiring
+import graphql.schema.idl.RuntimeWiring
+import graphql.validation.rules.ValidationRules
+import graphql.validation.schemawiring.ValidationSchemaWiring
+import org.springframework.beans.factory.ObjectProvider
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+
+@ConditionalOnClass(graphql.validation.rules.ValidationRules::class)
+@ConditionalOnProperty(
```

Comment:
I tend to agree, but would that make it inconsistent with the how extended-scalars or micrometer (any others???) are configured?

Reply:
This is a general approach to enable features in Spring Boot. For example, in the application I need some feature, but  in some tests for some specific reason it can be disabled with a property on tests.

Reply:
Yes, I agree that this is a general approach. However, for this particular feature, I didn't see a use case for having the module be present but disabled via a property. That said, I am fine with keeping it as is. 
@setchy - as you also pointed out, we do have the properties for those features that are already in separate modules. So I agree with your point about consistency as well.